src/components/elements/wysiwyg.tsx

```diff
@@ -52,6 +60,20 @@ const options: HTMLReactParserOptions = {
           )
 
         case "div":
+          if (nodeProps.className && nodeProps.className.includes("left-bar")) {
+            nodeProps.className = twMerge(
+              nodeProps.className,
+              "max-w-[725px] rs-pb-5 rs-pt-7 rs-pl-1 border-3 border-archway-dark border-opacity-50"
```

Comment:
Border should only be on the left.
![image](https://github.com/user-attachments/assets/1d29f93c-76a4-40c6-80d8-8e405220aec8)